### PR TITLE
Connect to api

### PR DESCRIPTION
**Pull request by rtonata88**
In this PR I have integrated the following Bookstore APIs for data preservation.

- Posting books
- Deleting books

### CODE REVIEW

File: src/redux/books/books.js [hunk 1]
```diff
@@ -13,12 +18,57 @@ export const removeBook = (payload) => ({
...
+      fetch(`${API_URL}/${action.payload.id}`, {
+        method: 'delete',
+        body: JSON.stringify({
+          item_id: action.payload.id,
+        }),
+        headers: {
+          Accept: 'application/json',
+          'Content-Type': 'application/json',
+        },
+      });
```
Comment (ZeenatLawal) [inline src/redux/books/books.js]: - [ ] I suggest the API functions should be created separately, that way the reducer would only return the state as expected. Kindly fix this.

File: src/redux/books/books.js [hunk 2]
```diff
@@ -13,12 +18,57 @@ export const removeBook = (payload) => ({
...
+        body: JSON.stringify({
+          title: action.payload.title,
+          item_id: action.payload.id,
+          category: action.payload.category,
+        }),
+        headers: {
+          Accept: 'application/json',
+          'Content-Type': 'application/json',
+        },
+      });
```
Comment (ZeenatLawal) [inline src/redux/books/books.js]: - [ ] I suggest the API functions should be created separately, that way the reducer would only return the state as expected. Kindly fix this.

**Review by ZeenatLawal — CHANGES_REQUESTED** [review]
Hi @rtonata88, 

Great job so far!:clap: :clap:
## To Highlight 🎊 🎊 
- Git flow is followed. ✔️
- PR has a title and summary. ✔️

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by DeliceLydia — APPROVED** [review]
# Congratulations🎉🎉

## Hi @rtonata88, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**